documents/openapi/openapi.yaml

```diff
@@ -1467,6 +1471,13 @@ paths:
           $ref: "#/components/responses/ServiceUnavailable"
     post:
       summary: Create an inbound number
+      tags:
+        - inbound-number
+      requestBody:
+        content:
+          application/json:
+            schema: 
+              $ref: "#/components/schemas/InboundNumberRequestData"
```

Comment:
this needs to have required fields as well

Reply:
added